README.md

```diff
@@ -11,6 +11,110 @@ go get github.com/golang/glog # Dependency
 go get github.com/moolitayer/awx-client-go/awx
 ```
 
+## Usage
+### import
+```go
+import 	"github.com/moolitayer/awx-client-go/awx"
+```
+
+### Creating a connection:
+```go
+// Uses the builder pattern:
+connection, err := awx.NewConnectionBuilder()
+  .Url("http://awx.example.com/api")          // Url is mandatory
```

Comment:
Note that the dots *must* be at the end of the lines, otherwise it will be a syntax error because the Go compiler automatically introduces a `;` at the end of the line if it doesn't end with that dot.

Reply:
oh right thanks